R/carehomes_rt.R

```diff
@@ -28,7 +28,13 @@ carehomes_Rt <- function(step, S, p) {
 
   calculate_ev <- function(t, S, drop_carehomes) {
     ## Next-Generation-Matrix
-    ngm <- beta[t] * outer(mean_duration[, t], S[, t]) * p$m
+    m <- p$m
+    ages <- seq_len(p$N_age - 2)
+    ch <- seq(to = p$N_age, length.out = 2)
```

Comment:
It's not immediately obvious what the above outputs (I thought it might be c(1, 19))
```suggestion
    ch <- c(p$N_age - 1, p$N_age)
```